com.microsoft.mrtk.graphicstools.unity/Runtime/Experimental/Magnifier/Shaders/Magnifier.shader

```diff
@@ -48,6 +49,7 @@ Shader "Graphics Tools/Magnifier"
             
             half MagnifierMagnification;
             float2 Center;
+            float4 _MousePos;
```

Comment:
Seems like `Center` and `_MousePos` are aliases for each other. Could we just use one variable instead? Maybe we call it `MagnifierCenter` since it is global to avoid collisions?